src/clog/TraceEmitterModules/CLogSTDOUT.cs

```diff
@@ -134,7 +134,7 @@ public void TraceLineDiscovered(string sourceFile, CLogOutputInfo outputInfo, CL
                         printf += "%S";
                         break;
                     case CLogEncodingType.Pointer:
-                        printf += "%p";
+                        printf += "0x%llx";
```

Comment:
But... doesn't this depend on pointer size?

Reply:
the worry is x86?  this should be okay no?